omnibus/package-scripts/agent-deb/prerm

```diff
@@ -158,6 +158,13 @@ remove_apm_remote_config()
     fi
 }
 
+remove_fips_module()
+{
+    # We explicitly remove the ssl directory because files within this folder are generated via a script
+    # outside of omnibus (omnibus only removes files present in the package).
```

Comment:
a bit of a nitpick, but it's the package itself that will only remove files it installed. Omnibus isn't at play once the package is generated